src/app/globals.css

```diff
@@ -1,3 +1,4 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700&display=swap");
```

Comment:
don't  use the font there are other approaches  https://nextjs.org/docs/pages/building-your-application/optimizing/fonts